### DISCUSSION TITLE
AV Effect

### DISCUSSION THREAD

**Discussion by smfthe1199** [ntsc-rs/ntsc-rs#103] [Presets]
Basically something like this, but moving!
![2024-05-07 00-51-27_ntsc](https://github.com/valadaptive/ntsc-rs/assets/153495179/a632f98d-915e-4799-936b-78f61c3956f7)

[AV_EFCT.json](https://github.com/valadaptive/ntsc-rs/files/15423840/AV_EFCT.json)

Do some modifications to the "Scanline Phrase Shift" and "Scanline Phrase Shift Offset" if **so desired**!

**Comment by smfthe1199** [comment]
If so desired, you can select "Alternating" in the "Use Field" Section to make it even more realistic, this makes a video issue in some tapes, where the use field highers in a frame, and then lowers in another frame, something like that!

**Reply by smfthe1199** [reply]
You can also make it "Both" in the "Use Field" section!

**Reply by Degamisu** [reply]
hey, i added on to your setting!

[AV_EFFECT2.json](https://github.com/valadaptive/ntsc-rs/files/15424950/AV_EFFECT2.json)

**Reply by smfthe1199** [reply]
Thanks!

**Reply by smfthe1199** [reply]
The reason for the _"Interleaved (upper first)_ option is because that it should be faster. And _Interleaved (Lower First)_ is the same but it begins with the lower part!

**Reply by smfthe1199** [reply]
Glad I came up with this in the first place!
